integration/mediation-tests/tests-mediator-1/src/test/resources/testng.xml

```diff
@@ -174,6 +174,10 @@
         <packages>
             <package name="org.wso2.carbon.esb.mediator.test.enrich"/>
         </packages>
+        <classes>
+            <class name="org.wso2.carbon.esb.mediator.test.enrich.jsonpath.EnrichIntegrationJsonPathTestCase">
```

Comment:
won't this be included in the package itself

Reply:
I tried. It didn't work.